troi/sorts.py

```diff
@@ -0,0 +1,34 @@
+from operator import attrgetter
+
+import troi
+from troi import PipelineError, Recording
+
+
+class YearSortElement(troi.Element):
+    '''
+        Sort recordings by year.
```

Comment:
add documentation that says behaviour is undefined for multiple items with the same year

Reply:
Done.